reconcile/database_access_manager.py

```diff
@@ -439,42 +442,52 @@ def _create_database_connection_parameter(
     oc: OCClient,
     admin_secret_name: str,
     user_secret_name: str,
-) -> DatabaseConnectionParameters:
+) -> dict[str, DatabaseConnectionParameters]:
```

Comment:
Another way is to create a new dataclass with 2 fields, or  typed dict to limit keys.

Reply:
Considered it, but since it's internal and just used right below it I omited that thought

Reply:
just for type check, still can be used as normal dict https://docs.python.org/3.8/library/typing.html#typing.TypedDict